src/custom-element.js

```diff
@@ -559,16 +559,20 @@ function createBaseCustomElementClass(win) {
      * Updates the layout box of the element.
      * See {@link BaseElement.getLayoutWidth} for details.
      * @param {!./layout-rect.LayoutRectDef} layoutBox
+     * @param {boolean} opt_hasMeasurementsChanged
```

Comment:
Also, is there a caller who doesn't pass a `hasMeasurementsChanged` param? If so, it's not really optional.

Reply:
yes just in tests, https://github.com/ampproject/amphtml/search?utf8=%E2%9C%93&q=updateLayoutBox&type=